charts/internal/control-plane/templates/auditlog-webhook-deployment.yaml

```diff
@@ -0,0 +1,124 @@
+# kind: Secret
+# metadata:
+#   name: splunk-audit-webhook-server
+#   namespace: kube-system
+# type: Opaque
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: splunk-audit-webhook
+  namespace: {{ .Release.Namespace }}
+  labels:
+    k8s-app: splunk-audit-webhook
+spec:
+  selector:
+    matchLabels:
+      k8s-app: splunk-audit-webhook
+  template:
+    metadata:
+      labels:
+        k8s-app: splunk-audit-webhook
+        app: splunk-audit-webhook
+        networking.gardener.cloud/to-dns: allowed
+        networking.gardener.cloud/to-public-networks: allowed
+      annotations:
+          scheduler.alpha.kubernetes.io/critical-pod: ''
+        spec:
+      # it's better to disable the service links as the default args do not work properly otherwise (metal#64)
```

Comment:
Unfortunately we do not have metal#64 anymore because we went open source and this still points to our old Gitlab issues. Maybe remove or re-formulate.

Reply:
`enableServiceLinks: False` is not in the other deployments (any more?), so removing it here too.